ena_upload/templates/ENA_template_vir_sample.xml

```diff
@@ -29,6 +29,26 @@
                 <TAG>geographic location (country and/or sea)</TAG>
                 <VALUE>${row.geographic_location}</VALUE>
             </SAMPLE_ATTRIBUTE>
+            <py:if test="hasattr(row, 'geographic_location_region')">
+                <SAMPLE_ATTRIBUTE>
+                    <TAG>geographic location (region and locality)</TAG>
+                    <VALUE>${row.geographic_location_region}</VALUE>
+                </SAMPLE_ATTRIBUTE>
+            </py:if>
+            <py:if test="hasattr(row, 'geographic_location_latitude')">
+                <SAMPLE_ATTRIBUTE>
+                    <TAG>geographic location (latitude)</TAG>
+                    <VALUE>${row.geographic_location_latitude}</VALUE>
+                    <UNITS>DD</UNITS>
+                </SAMPLE_ATTRIBUTE>
+            </py:if>
+            <py:if test="hasattr(row, 'geographic_location_longitude')">
+                <SAMPLE_ATTRIBUTE>
+                    <TAG>geographic location (longitude)</TAG>
+                    <VALUE>${row.geographic_location_longitude}</VALUE>
+                    <UNITS>DD</UNITS>
+                </SAMPLE_ATTRIBUTE>
+            </py:if>          
```

Comment:
I did live testing and I _was able to upload my reads to ENA with **only** mandatory metadata in samples table_ after adding `hasattr` test. I think that latitude and longitude can be evaluated together, like so:

```suggestion
            <py:if test="hasattr(row, 'geographic_location_latitude') and hasattr(row, 'geographic_location_longitude')">
                  <SAMPLE_ATTRIBUTE>
                      <TAG>geographic location (latitude)</TAG>
                      <VALUE>${row.geographic_location_latitude}</VALUE>
                      <UNITS>DD</UNITS>
                  </SAMPLE_ATTRIBUTE>
                  <SAMPLE_ATTRIBUTE>
                      <TAG>geographic location (longitude)</TAG>
                      <VALUE>${row.geographic_location_longitude}</VALUE>
                      <UNITS>DD</UNITS>
                  </SAMPLE_ATTRIBUTE>
              </py:if>  
```

Testing if value is not `None` can be also added. Currently, script fails later if key is present but the value is missing, so that samples with different metadata completeness should be uploaded separately.